provider/maas/add-juju-bridge.py

```diff
@@ -346,21 +346,22 @@ def main(args):
         if not os.path.isfile(backup_file):
             shutil.copy2(args.filename, backup_file)
 
-    ifquery = "$(ifquery --interfaces={} --exclude=lo --list)".format(args.filename)
-
     print("**** Original configuration")
     print_shell_cmd("cat {}".format(args.filename))
     print_shell_cmd("ifconfig -a")
-    print_shell_cmd("ifdown --exclude=lo --interfaces={} {}".format(args.filename, ifquery))
 
     print("**** Activating new configuration")
-
     with open(args.filename, 'w') as f:
         print_stanzas(stanzas, f)
         f.close()
 
+    # --no-loopback isn't supported on precise so use --exclude=lo instead.
```

Comment:
Please format with pyfmt the script with gofmt. I would also say that if this change is strictly not necessary and ifquery works then don't change it. There's too much manual effort required to test all variations.